frontend/public/components/container.jsx

```diff
@@ -108,6 +108,21 @@ const Env = ({env}) => {
   </table>;
 };
 
+// Split image string into the Docker image string and tag (aka version) portions. The tag defaults to 'latest'.
+const getContainerImageAndTag = image => {
+  if (_.isEmpty(image)) {
+    return [null, null];
+  }
+  const semiColonSplit = image.split(':');
+  if (_.size(semiColonSplit) === 1) {
+    return [image, 'latest'];
+  }
+  const imageTag = _.last(semiColonSplit);
+  const imageNamePortion = _.trim(semiColonSplit.slice(-2, -1), '@sha256');
+  const imageName = imageNamePortion.split('/').slice(-2).join('/');
+  return [imageName, imageTag];
+};
```

Comment:
Contemplating an object `{ image: '', tag: '' }` vs array/tuple, but otherwise looks good to me!